### PR TITLE
chore(moderation: add note about spam posts

### PR DESCRIPTION
**Pull request by mmarchini**

### CODE REVIEW

**Comment by mmarchini** [comment]
cc @nodejs/tsc @nodejs/community-committee @nodejs/moderation

**Review by ljharb — APPROVED** [review]
LGTM (moderation team)

**Review by targos — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by bnb — APPROVED** [review]

**Review by cjihrig — APPROVED** [review]

File: Moderation-Policy.md [hunk 1]
```diff
@@ -202,6 +202,8 @@ of the [Code of Conduct][].
   * report accounts to GitHub
 * Accounts that are reasonably believed to be bots (other than bots authorized
   by both TSC and CommComm) are subject to immediate Blocking.
+* Issues, pull requests, discussions and comments that are considered spam by the Moderation Team
+  (such as job posting, service advertising, etc.) will be deleted.
```
Comment (Trott) [inline Moderation-Policy.md:206]: For consistency with the other items in the list:

```suggestion
* Issues, pull requests, discussions, and comments that are spam (job posting,
  service advertising, etc.) are subject to Immediate moderation.
```

**Review by Trott — APPROVED** [review]
LGTM with or without my suggested wording change.

File: Moderation-Policy.md (hunk 1 above)
Comment (nschonni) [reply Moderation-Policy.md:206]: lower case "immediate"?

**Review by danielleadams — APPROVED** [review]

File: Moderation-Policy.md (hunk 1 above)
Comment (Trott) [reply Moderation-Policy.md:206]: > lower case "immediate"?

Yes. Fixed. Thanks!

**Comment by mhdawson** [comment]
@mmarchini are you ok with @Trott's suggestion. If so I can accept them and land this (or you can do that too:))

**Pull request merged**